cli.js

```diff
@@ -24,12 +24,12 @@ var p = new HalModuleParser();
 
 var args = process.argv;
 if (args.length <= 2) {
-	console.log("no filename specified");
+	console.log('no filename specified');
 	process.exit(-1);
 }
 var filename = process.argv[2];
 
 
 p.parseFile(filename, function() {
-	console.log("got ", arguments);
-});
\ No newline at end of file
+	console.log('got', arguments);
```

Comment:
Space is automatically added if you separate arguments instead merging them into one string: